### ISSUE TITLE
gesture request: double tap

### ISSUE THREAD

**Issue by maoamid** [eclipsesource/tabris-js#1172]
Is it hard to support  double tap gesture?

**Comment by cookieguru** [comment]
No, you'll just need to listen for the `tap` event.  If another `tap` event is fired again, and it is within a duration that you decide, you can determine it was a double tap.  Pseudocode:

```plain
last_timestamp = null
widget.on(tap) => {
	if(last_timestamp is not null and now - last_timestamp < 500) {
		// double tap
	}
	last_timestamp = now
}

**Comment by mpost** [comment]
The solution @cookieguru proposed will actually work but in general we should add support for it in Tabris.js

**Comment by cookieguru** [comment]
@mpost that would mean the timeout is platform dependent?

**Comment by mpost** [comment]
@cookieguru It would follow the platform conventions.
